File: app/src/main/java/com/dakota/gallimore/portfolio/Education.java
package com.dakota.gallimore.portfolio;

import com.thoughtbot.expandablerecyclerview.models.ExpandableGroup;

import java.util.List;

/**
 * Created by Dakota on 10/26/2017.
 */

public class Education extends ExpandableGroup<ChildEducation> {

    public Education(String title, List<ChildEducation> items) {
        super(title, items);
    }


}

File: app/src/main/java/com/dakota/gallimore/portfolio/Education.kt
package com.dakota.gallimore.portfolio

import com.thoughtbot.expandablerecyclerview.models.ExpandableGroup

/**
 * Created by Dakota on 10/26/2017.
 */

class Education(title: String, items: List<ChildEducation>) : ExpandableGroup<ChildEducation>(title, items)
